src/UKHO.ERPFacade.Common/UKHO.ERPFacade.Common.csproj

```diff
@@ -30,7 +30,9 @@
     <PackageReference Include="Microsoft.Extensions.Options" Version="7.0.1" />
     <PackageReference Include="Azure.Data.Tables" Version="12.8.3" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
+    <PackageReference Include="SharpZipLib" Version="1.4.2" />
```

Comment:
Is this package required?

Reply:
Yes

Reply:
this package is used in permit decryption logic.